Tests/NIOTests/NonBlockingFileIOTest.swift

```diff
@@ -452,6 +452,45 @@ class NonBlockingFileIOTest: XCTestCase {
         }
         XCTAssertEqual(2, numCalls)
     }
+    
+    func testReadingFileSize() throws {
+        try withTemporaryFile(content: "0123456789") { (fileHandle, _) -> Void in
+            let size = try self.fileIO.readFileSize(fileHandle: fileHandle,
+                                                    eventLoop: eventLoop).wait()
+            XCTAssertEqual(size, 10)
+        }
+    }
+    
+    func testChangeFileSizeShrink() throws {
+        try withTemporaryFile(content: "0123456789") { (fileHandle, _) -> Void in
+            try self.fileIO.changeFileSize(fileHandle: fileHandle,
+                                           size: 1,
+                                           eventLoop: eventLoop).wait()
+            let fileRegion = try FileRegion(fileHandle: fileHandle)
+            var buf = try self.fileIO.read(fileRegion: fileRegion,
+                                           allocator: allocator,
+                                           eventLoop: eventLoop).wait()
+            XCTAssertEqual("0", buf.readString(length: buf.readableBytes) ?? "bad")
```

Comment:
No need to nil-coalesce here: Swift will automatically promote `"0"` to `Optional<String>.some("0")`.